### ISSUE TITLE
The training loss implementation for the vision tower.

### ISSUE THREAD

**Issue by stdKonjac** [mit-han-lab/vila-u#21]
Hi, thanks for your great work! 
As far as I know, the released code only includes the quantization loss, while the reconstruction loss is missing. And I did not find a detailed description of it in the paper. Could you please share the implementation of the training loss mentioned in the paper? @zhuoyang20
